### DISCUSSION TITLE
Can I revalidate on firebase ?

### DISCUSSION THREAD

**Discussion by watanabeyu** [vercel/next.js#15671] [Help]
I use SSG(Nextjs v9.5) on firebase hosting, and configuring `fallback:true` with `revalidate:1`.
But can't revalidate files, so that these errors happend.

<img width="929" alt="スクリーンショット 2020-07-30 14 48 35" src="https://user-images.githubusercontent.com/6455904/88885599-c6680800-d273-11ea-9ad4-0bb9f374e056.png">

SSG is nice solution, so I want to use revalidate on firebase.
I would like to know how to get around these.

thanks.

**Comment by leerob** [comment]
I'm not an expert on Firebase Hosting, but I don't believe there's a Next.js adapter for ISR and their hosting platform. They would need to add that (or something open source) so that the Firebase CDN understands how to purge their cache. 

I'm biased, but give Vercel a shot. Works out of the box.

https://vercel.com/import

**Reply by FSaldanha** [reply]
I'm not 100% sure, but I did some testing and it looks like Firebase supports cache settings for differents forms of Next.js pre-rendering.

If someone could check it out more closely, here are the links:
Demo: https://teste-next-8ea5e.web.app/
Source: https://github.com/FSaldanha/next-firebase-test
